clients/src/test/java/org/apache/kafka/test/TestSslUtils.java

```diff
@@ -116,11 +115,8 @@ private static KeyStore createEmptyKeyStore() throws GeneralSecurityException, I
 
     private static void saveKeyStore(KeyStore ks, String filename,
                                      Password password) throws GeneralSecurityException, IOException {
-        FileOutputStream out = new FileOutputStream(filename);
-        try {
+        try (FileOutputStream out = new FileOutputStream(filename)) {
```

Comment:
This is using the "try-with-resources statement" so it will take care of the close.